BUILD.gn

```diff
@@ -838,6 +838,7 @@ group("gn_all") {
       deps += [ "//components/cronet:cronet_tests" ]
     }
   }
+  deps += [ "//chrobalt:gn_all" ]
```

Comment:
Should this be inside of `if (chrobalt)` block ?

Reply:
done